Makefile-fuzz.am

```diff
@@ -46,5 +46,11 @@ FUZZ = $(check_PROGRAMS)
 
 DISTCLEANFILES += Makefile-fuzz-generated.am
 
+# Non Autogenerated Fuzz Tests
+noinst_PROGRAMS += test/fuzz/tcti-spi-helper-fuzz-test
+test_fuzz_tcti_spi_helper_fuzz_test_CFLAGS         = -I bill $(FUZZ_CFLAGS) $(FUZZ_CFLAGS) $(AM_CFLAGS)
```

Comment:
@williamcroberts Can -I bill be removed?

Reply:
> @williamcroberts Can -I bill be removed?

Hahaha debugging the makefile, but I think it should stay 😂